test/manager/bazel/_fixtures/repositories.bzl

```diff
@@ -0,0 +1,229 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Please add some tests to `update.spec.js` that reference this fixture

Reply:
added test to the `test/manager/bazel/update.spec.js`